tests/tests_suite_test.go

```diff
@@ -86,3 +93,11 @@ func getMaxFailsFromEnv() int {
 
 	return maxFails
 }
+
+// Collect info directly after each `It` execution with our reporters
+// to collect the state directly after the spec.
+var _ = JustAfterEach(func() {
```

Comment:
Probably there is some confusion in the PR description and commit message:
> Collect reporter artifacts in a Just**Before**Each

but maybe should be

> Collect reporter artifacts in a Just**After**Each

Reply:
Great catch. Fixed in the commit and the PR description.